### ISSUE TITLE
4.0.0: Fatal error when /etc/n98-magerun2.yaml exists

### ISSUE THREAD

**Issue by jeroenvermeulen** [netz98/n98-magerun2#516]
Steps to reproduce:
- Upgrade to 4.0.0
- `/usr/local/bin/n98-magerun2.phar` => works
- `sudo touch /etc/n98-magerun2.yaml`
- `sudo chmod 644 /etc/n98-magerun2.yaml`
- `/usr/local/bin/n98-magerun2.phar` => Error:

```
PHP Fatal error:  Uncaught TypeError: Argument 2 passed to N98\Util\ArrayFunctions::mergeArrays() must be of the type array, string given, called in phar:///usr/local/bin/n98-magerun2.phar/src/N98/Magento/Application/ConfigurationLoader.php on line 193 and defined in phar:///usr/local/bin/n98-magerun2.phar/src/N98/Util/ArrayFunctions.php:27
Stack trace:
#0 phar:///usr/local/bin/n98-magerun2.phar/src/N98/Magento/Application/ConfigurationLoader.php(193): N98\Util\ArrayFunctions::mergeArrays(Array, '/etc/n98-mageru...')
#1 phar:///usr/local/bin/n98-magerun2.phar/src/N98/Magento/Application/ConfigurationLoader.php(114): N98\Magento\Application\ConfigurationLoader->loadSystemConfig(Array)
#2 phar:///usr/local/bin/n98-magerun2.phar/src/N98/Magento/Application/Config.php(257): N98\Magento\Application\ConfigurationLoader->getPartialConfig(true)
#3 phar:///usr/local/bin/n98-magerun2.phar/src/N98/Magento/Application.php(380): N98\Magento\Application\Config->loadPartialConfig(true)
#4 phar:///usr/local/bin/n98-magerun2.phar/src/N98/Magen in phar:///usr/local/bin/n98-magerun2.phar/src/N98/Util/ArrayFunctions.php on line 27
```

I also tried using the [repo Yaml](https://github.com/netz98/n98-magerun2/blob/develop/config.yaml) config as `/etc/n98-magerun2.yaml` but the same error occurs.

**Comment by cmuench** [comment]
@jeroenvermeulen Thanks for reporting. Hotfix will be deployed in the next hour.

**Comment by cmuench** [comment]
Fix is deployed as 4.0.1.
